### PR TITLE
implemented Session.to_pickle

### PR DESCRIPTION
**Pull request by gdementen**
also contains other misc improvements

### CODE REVIEW

File: doc/source/changes/version_0_22.rst.inc [hunk 1]
```diff
@@ -52,12 +58,26 @@
...
+
     >>> arr2 = ndtest((3, 3))
-    >>> arr.reindex(arr2.axes, fill_value=-1)
+    >>> arr2
+    a\\b | b0 | b1 | b2
+     a0 |  0 |  1 |  2
+     a1 |  3 |  4 |  5
+     a2 |  6 |  7 |  8
+    >>> arr.reindex(arr2.axes, fill_value=0)
     a\\b | b0 | b1 | b2
```
Comment (gdementen) [inline doc/source/changes/version_0_22.rst.inc]: oups :) I need to change this :)

File: larray/tests/test_session.py [hunk 2]
```diff
@@ -142,40 +138,57 @@ def test_h5_io(self):
...
+        # CSV cannot keep ordering (so we always sort keys)
+        self.assertEqual(list(s.keys()), ['e', 'f', 'g'])
+
+    def test_pickle_io(self):
+        fpath = abspath('test_session.pkl')
+
+        self.session.save(fpath)
+        s = Session()
+        s.load(fpath, engine='pickle')
+        self.assertEqual(list(s.keys()), ['e', 'g', 'f'])
```
Comment (alixdamman) [inline larray/tests/test_session.py]: So here the saved/loaded session contains only LArray objects?

File: doc/source/changes/version_0_22.rst.inc [hunk 3]
```diff
@@ -1,8 +1,14 @@
 ﻿New features
 ------------
 
-* implemented a new describe() method on arrays to give quick summary statistics (excluding NaN values). By default,
-  it includes the number of values, mean, standard deviation, minimum, 25, 50 and 75 percentiles and maximum.
+* viewer: added a menu bar with the ability to clear the current session, save all its arrays to a file (.h5, .xlsx,
+  .pkl or a directory containing multiple .csv files), and load arrays from such a file (closes :issue:`88`).
+
```
Comment (gdementen) [inline doc/source/changes/version_0_22.rst.inc]: no, it **currently** contains only the arrays, like other formats.

File: larray/tests/test_session.py (hunk 2 above)
Comment (gdementen) [inline larray/tests/test_session.py]: indeed

**Comment by gdementen** [comment]
I now know the way I want to fix #241, but I am unsure we should do it for this release.

**Comment by alixdamman** [comment]
OK, #241 will be fixed in the next release (0.23)

**Pull request merged**